src/Ztm.Data.Entity/Contexts/Main/ExodusPayload.cs

```diff
@@ -0,0 +1,12 @@
+using NBitcoin;
+
+namespace Ztm.Data.Entity.Contexts.Main
+{
+    public sealed class ExodusPayload
+    {
+        public uint256 TransactionHash { get; set; }
+        public Script Sender { get; set; }
```

Comment:
We don't need `Sender` to be `Script` here due to Exodus only working with the address, not script.

Reply:
Change it to string and store bitcoin address instead